src/core/export.ts

```diff
@@ -60,7 +63,7 @@ module.exports = function() {
 		return fetch(url, opts);
 	};
 
-	app.use(sapper());
+	app.use(require('./middleware')()); // TODO this is filthy
```

Comment:
Looks like this missed a level, should be `../middleware`.

Reply:
it's meant to be this way in fact — the built files are all in the project root (`/core.js`, `/middleware.js`), so while it looks incorrect from the perspective of the source code, it's correct from the perspective of the built code. Ideally it would `import` the middleware but it was finicky so I cheated, hence the TODO

Reply:
Got it 👍 On a more recent pass I realized that & forgot to remove comment~